index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
```

Comment:
In that case, how many H.264 codec capabilities do you get? One per supported profile (which I believe is what SDP requires), or only one per supported profile that's not a subset of another supported profile? For example, if a browser supports constrained baseline through extended, does it only put extended in the capabilities?

And if "configured identically between sender and receiver" doesn't work due to this... What would be better wording? "One capability for each non-interchangeable configuration of a codec?" Or maybe it would be more clear if each codec parameter's documentation mentioned whether or not it creates distinct codec capability entries.